.github/workflows/release.yml

```diff
@@ -37,7 +40,7 @@ jobs:
           node-version: 18
       - run: npm install
       - run: npm run build
-      - run: npm run test:integ
+      - run: npm run test:integ -- --shard=${{ matrix.shard }}/${{ strategy.job-total }}
```

Comment:
should this be reflected in the current check for this PR? https://github.com/cloudscape-design/components/actions/runs/11019134425/job/30601298161#step:5:2 ?

Reply:
The dry-run actions are covered here: https://github.com/cloudscape-design/actions/pull/42